OpenSim/Common/ComponentConnector.h

```diff
@@ -265,13 +274,18 @@ class  Input : public AbstractInput {
         AbstractInput(name, connectAtStage) {}
 
     /** Connect this Input the from provided (Abstract)Output */
-    void connect(const AbstractOutput& output) const override{
+    void connect(const AbstractOutput& output) const override {
         // enable interaction through AbstractInterface
         Super::connect(output);
         // and value specific interface
         connectee = Output<T>::downcast(output);
     }
 
+    /** Connect this Input given a root Component to search for
+        the Output according to the connect_to_name of this Input  */
+    void findAndConnect(const Component& root) override {
+    }
```

Comment:
It doesn't, yet. I couldn't leave it unimplemented, since Input cannot be abstract. It should have a proper implementation that finds the Output<T> to connect to.